server/server.js

```diff
@@ -24,6 +24,8 @@ mongoose.connect(
   }
 );
 mongoose.set("useCreateIndex", true);
+// Fix deprecation warning for useFindOneandUpdate instead of useFindAndModify
+mongoose.set("useFindAndModify", false);
```

Comment:
Cool, I probably accidentally overwrote this on the last one 😅

Reply:
This was a new add 👍